### DISCUSSION TITLE
为什么电脑提示下载的文件有病毒?

### DISCUSSION THREAD

**Discussion by JamesFu7771** [leiurayer/downkyi#357] [Q&A]
为什么我的2台电脑都提示有病毒？ 1个是360杀毒，另外1个是微软自带的提示“检测到病毒
![图片_20220427180445](https://user-images.githubusercontent.com/65690729/165639341-95ca5906-84a1-480c-96a1-a53581708500.png)
)
”

**Comment by leiurayer** [comment]
几种解决方案：

1. 添加白名单
2. 沙盒运行
3. 虚拟机运行
4. 自己编译代码
